public/Lab-Files/Lesson04/Assignments/public2/yellow.html

```diff
@@ -0,0 +1,65 @@
+    <link rel="stylesheet" href="css/styles.css" type="text/css"/>
+</head>
+
+<body>
+    <h1>Recent From Blog</h1>
+    <nav class="nav"><p class="intro">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Lorem ipsum dolor</p></nav>
+    <main>
+        <article>
+            <section class="section1">
+                <img class="img1" src="img/keyboard.JPG" alt="hands typing on keyboard">
+                <p class="lefttext">Lorem ipsum dolor sit amet, consectetur adipisicing elit. At, inventore.</p>
+                <p class="read"><a href="readmore.com">Read More</a></p>
+            </section>
+        </article>
+        <div class="rightbox">
+            <div class="box1">
+                <img class="img2" src="img/laptop.jpg" alt="laptop and headphones sitting on a bed"> 
+                <p class="righttext">Lorem ipsum dolor sit amet, aperiri assueverit nam ut, scaevola singulis in cum.</p>
+            </div>
+            <div class="box2">
+                <img class="img3" src="img/desk.jpg" alt="laptop and mouse on desk">
+                <p class="righttext">Lorem ipsum dolor sit amet, aperiri assueverit nam ut, scaevola singulis in cum.</p>
+            </div>
+            <div class="box3">
+                <img class="img4" src="img/window.jpg" alt="man staring out window">
+                <p class="righttext">Lorem ipsum dolor sit amet, aperiri assueverit nam ut, scaevola singulis in cum.</p>
+            </div>
+        </div>
+    </main>
+        <hr>
+    <footer>
+        
+        <div class="foot1"><h4>Why Profession?</h4>
+        <div class="profession"><img src="img/green_checkmark_small.png" alt="green checkmark" class="check"><p class="clean">Clean & Corporate webdesign</p>
+        </div>
+        <div class="features"><img src="img/green_checkmark_small.png" alt="green checkmark" class="check"><p class="template">Well coded HTML template</p>
+        </div>
+        <div class="pricing"><img src="img/green_checkmark_small.png" alt="green checkmark" class="check"><p class="layout">Multiple Layout Options</p>
+        </div>
+        </div><hr>
+            
+        <div class="foot2"><h4>Main Features</h4>
+        <img src="img/green_checkmark_small.png" alt="green checkmark" class="check"><p class="boot">Bootstrap 3.x & SASS</p>
+        <img src="img/green_checkmark_small.png" alt="green checkmark" class="check"><p class="speed">Super lightning page speed</p>
+        <img src="img/green_checkmark_small.png" alt="green checkmark" class="check"><p class="doc">Well documented code</p>
+        </div><hr>
+        
+        <div class="foot3"><h4>Transparent Pricing</h4>
+        <p>Are you interested in our services? don't hesitate, contact us</p>
+    
+        </div><hr>
+            
+     
+    </footer>
+    <hr>
+    <script type="text/javascript" src="js/script.js"></script>
+</body>
+
+</html>
```

Comment:
You did a great job here, it looks very similar to the actual design! I think you could try nesting your divs and the content, so that it is easier to read and keep track of. Something similar to this:
```
<div class="example">
    <img src="#" alt="example">
    <p>Text here</p>
</div>
```